AmethystAPI/src/minecraft/src/common/world/level/block/Block.cpp

```diff
@@ -45,6 +45,7 @@ gsl::strict_not_null<const Block*> Block::setState(const BlockState& stateType,
 
 template gsl::strict_not_null<const Block*> Block::setState<int>(const BlockState& stateType, int value) const;
 
+// Just a partial reimplementation using `BlockLegacy::getMapColor`
```

Comment:
For clarity also specify the version e.g. `// 1.20.71.1 - Partial reimplementation`